examples/vnet/kubernetesvnet.json.pretest.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+az network vnet create -g ${RESOURCE_GROUP} -n KubernetesCustomVNET --address-prefix 10.239.0.0/16 --subnet-name KubernetesSubnet --subnet-prefix 10.239.0.0/16
```

Comment:
The scripts for DCOS and SwarmMode are identical, but for k8s is slightly different.

The reason for duplication is to make the pre-test generic. If the "api-model".json.pretest.sh exists, we will execute it.

I'm open to other suggestions.